### PR TITLE
fix: Fix unsuitable behaviors

### PR DESCRIPTION
**Pull request by npofsi**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,10 +3,16 @@ module github.com/beyondstorage/go-service-ftp
 go 1.15
 
 require (
-	github.com/beyondstorage/go-credential v0.0.0-20210804082910-7c63d94f2826
+	github.com/beyondstorage/go-credential v0.0.0-20210918043243-9d146b959702
```
Comment (Xuanwo) [inline go.mod]: Please use stable release of go-credential: https://github.com/beyondstorage/go-credential/releases/tag/v0.1.0

**Comment by Xuanwo** [comment]
Hi, Build Test reports an error here. Maybe you need to run `make build` to make sure everything has been updated.

**Comment by npofsi** [comment]
Oh! I forget to generate files.

**Comment by npofsi** [comment]
I have run `make build` and regenerate the files, but no file changed.

**Comment by Xuanwo** [comment]
Ooops, I can see `go.sum` has been changed. Which go version you used locally?

**Comment by npofsi** [comment]
Oops! I just install go with `apt install go` when I move to new laptop, and it is 1.13.8

**Comment by npofsi** [comment]
Now I should install latest go manually.

**Comment by Xuanwo** [comment]
Oh, it's so old. How about upgrading to go 1.15 or higher?

**Comment by npofsi** [comment]
ok

File: go.mod [hunk 2]
```diff
@@ -3,10 +3,16 @@ module github.com/beyondstorage/go-service-ftp
 go 1.15
 
 require (
-	github.com/beyondstorage/go-credential v0.0.0-20210804082910-7c63d94f2826
+	github.com/beyondstorage/go-credential v0.1.0
 	github.com/beyondstorage/go-endpoint v1.1.0
 	github.com/beyondstorage/go-integration-test/v4 v4.5.0
 	github.com/beyondstorage/go-storage/v4 v4.7.0
+	github.com/gopherjs/gopherjs v0.0.0-20210923143318-357ed63a84fb // indirect
```
Comment (Xuanwo) [inline go.mod]: Why do we need these modules? Can we just remove them?

**Comment by Xuanwo** [comment]
So nice!

**Pull request merged**